From 8e679bb1f5ed9a4697b474afeb6ee84f8c7e2644 Mon Sep 17 00:00:00 2001
From: David Bauer <mail@david-bauer.net>
Date: Tue, 24 Aug 2021 19:49:04 +0200
Subject: [PATCH] modules: update OpenWrt

9d082556fe kernel: bump to 4.14.244

Signed-off-by: David Bauer <mail@david-bauer.net>
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 2424122d..65e3bb23 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='packages routing gluon'
 
 OPENWRT_REPO=https://github.com/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-19.07
-OPENWRT_COMMIT=7378c949270067d0a633cfd892b9dd6416581afa
+OPENWRT_COMMIT=9d082556fecfe2e971f964ebcd6b13fe30d208af
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-19.07
-- 
2.17.1

