From 386fd5b1011a2e2e47a99a51f5a8f0e7913ec70b Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Thu, 21 Apr 2022 17:50:33 +0200
Subject: [PATCH] modules: update OpenWrt packages

1c5e4c80f49b zabbix: update to version 4.0.37
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index abd47fc5..c0940352 100644
--- a/modules
+++ b/modules
@@ -6,7 +6,7 @@ OPENWRT_COMMIT=ecbbb373edf7be017e546be2443e6c422cb9c220
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-19.07
-PACKAGES_PACKAGES_COMMIT=5a842639dc87dab9daab34c064f6c8e268a34ba8
+PACKAGES_PACKAGES_COMMIT=1c5e4c80f49bfddaee1998636fd8efe915fee7fc
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt/routing.git
 PACKAGES_ROUTING_BRANCH=openwrt-19.07
-- 
2.17.1

